PAN-OS Integration - New Command Bugfixes (#20437)
demisto/content: Packs/PAN-OS/ReleaseNotes/1_14_14.md

#### Integrations
##### Palo Alto Networks PAN-OS

> - Fixed issue when Panorama systems are used in target arguments for certain commands
> - Added `LogForwardingProfile` to list of objects supported by `pan-os-config-get-objects`

Reviewer: ```suggestion
- Fixed an issue when Panorama systems are used in target arguments for certain commands.
```
